test/check_git_size.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+# Check .git log size
+# Over time, this will increase s.t. the value will need to be adjusted.
```

Comment:
Maybe helpful: https://stackoverflow.com/questions/10845051/git-show-total-file-size-difference-between-two-commits/10847242
Also okay to keep as is...

Reply:
looks good, I think I will leave it as is for the moment though.